Comments/ClientApp/src/components/App/App.scss

```diff
@@ -0,0 +1,12 @@
+// TODO: consider feeding these changes back to the design system
```

Comment:
i agree with this comment. these changes look like they belong in the design system.

Reply:
There are a couple of bits I've already found that need to be fed back up into the design system so I'll make a note.